src/firefly_exchange_client/client.py

```diff
@@ -384,39 +390,41 @@ async def withdraw_margin_from_bank(self, amount):
 
         return True;
 
-    async def adjust_leverage(self, symbol, leverage):
+    async def adjust_leverage(self, symbol, leverage, parentAddress:str=""):
         """
             Adjusts user leverage to the provided one for their current position on-chain and off-chain.
             If a user has no position for the provided symbol, leverage only recorded off-chain
 
             Inputs:
                 symbol (MARKET_SYMBOLS): market for which to adjust user leverage
                 leverage (number): new leverage to be set. Must be in base decimals (1,2 etc.)
-
+                parentAddress (str): optional, if provided, the leverage of parent is 
```

Comment:
Does this mean the subaccount can change the leverage of the parent account? Is that behavior possible?

Reply:
Yes this is possible. The sub account does not have its own position, its trading on a parent's position. Just like it can trade on parent position, it can also adjust its margin and leverage